atomicapp/utils.py

```diff
@@ -360,13 +360,17 @@ def getUniqueUUID():
         return data
 
     @staticmethod
-    def loadAnswers(answers_file):
+    def loadAnswers(answers_file, format=None):
         if not os.path.isfile(answers_file):
             raise AtomicAppUtilsException(
                 "Provided answers file does not exist: %s" % answers_file)
 
         logger.debug("Loading answers from file: %s", answers_file)
-        return anymarkup.parse_file(answers_file)
+        try:
+            result = anymarkup.parse_file(answers_file, format=format)
+        except anymarkup.AnyMarkupError:
+            result = anymarkup.parse_file(answers_file)
+        return result
```

Comment:
so, when we have ``:`` or ``.`` in the section names in an ``INI`` file, anymarkup does not identify it as an INI file by default, so, we have to explicitly mention the fromat when parsing the file. This L370 also works well if the user specifies a non default answers format, as well.

L372 is to handle the scenario when the user points to a JSON/YAML answers file but does not set the answers format, which defaults to ``INI``. In such a case, L370 won't be able to load the file inspite of the fact that the file is a proper JSON, YAML file.